spec/features/collection_drafts/collection_draft_creation_spec.rb

```diff
@@ -54,6 +54,16 @@
             expect(page).to have_content('123')
           end
         end
+
+        context 'when accessing a collection draft\'s json' do
+          before do
+            visit collection_draft_path(CollectionDraft.first, format: 'json')
+          end
+
+          it 'returns the json' do
+            expect(page).to have_content('123')
```

Comment:
wonder if you should check for a json tag here rather than just content?

Reply:
I certainly accede to the argument that I should check for a json tag.  My question is if there is value in checking that the content is as expected or if we trust the content is verified elsewhere.

Reply:
I'm inclined to think a quick check that a json tag exists, as the purpose of the ticket is produce some kind of json output, seems a check with that regard would be useful in case something changes later and breaks it.

Reply:
Selenium doesn't support checking the header tags.  I have updated the tests so that they are actually differentiating between what they expect from an html response versus a json response.  As Thomas described it, it's duck type testing.  I do not think this satisfies the heart of your request, but I think this is the best available compromise.